src/controllers/db/cities.js

```diff
@@ -0,0 +1,19 @@
+import {getModels, getURLParam} from './utils';
+
+export const queries = {
+    postgres: {},
+    mongo: {
+        getCities: req => getModels(req).City.find({}),
+        addCity: req => getModels(req).City.create(req.body),//then(instance => instance.validate()),
+        updateCity: req => getModels(req).City.findOneAndUpdate({
+            _id: getURLParam(req, 'id')
+        }, req.body, {
```

Comment:
what's inside body? How I can understand what you insert here, go to client side code?:)